packages/cli/lib/lib/webpack/webpack-client-config.js

```diff
@@ -109,6 +109,10 @@ async function clientConfig(env) {
 						from: resolve(__dirname, '../../resources/sw-debug.js'),
 						to: 'sw-debug.js',
 					},
+					existsSync(source('static/root')) && {
```

Comment:
it does not need to be in `static/root` it was an option to pic a name. just pick any one.